docs/UserGuide.adoc

```diff
@@ -200,7 +200,12 @@ Multiple steps and multiple ingredients for a recipe can be added in this one co
 {nbsp} +
 The compulsory fields needed in a recipe are the *name, time and at least one of the ingredient fields.*
 You can add these fields first, then edit the recipe as needed using the various edit commands available
-(Refer to section 6.6 and 7).
+(Refer to section 6.6 and 7). +
+{nbsp} +
+For name and time fields that only require one input, if more than one name or more than one time is given,
```

Comment:
Will it be better if we split this new paragraph into a new [TIP]? (this questions applies to the `Edit` command as well! I'm asking because I'm just concerned that this block might be too chunky. But if you guys think it's fine then all's good! 😄

Add:
![add](https://user-images.githubusercontent.com/57068044/78739040-fe2a0780-7985-11ea-8c7c-1b9de52fb81b.PNG)

Edit:
![edit](https://user-images.githubusercontent.com/57068044/78739199-54974600-7986-11ea-8b42-2a73f9ffedda.PNG)

Reply:
i think spliting is a good idea too! Thanks for the hardwork!!

Reply:
Thats true! Ill split it up! Thanks for the suggestions